timbermill-python/timbermill/timberlog_event_handler.py

```diff
@@ -30,7 +31,7 @@
 drain_events_from_queue = None
 
 
-def init(timbermill_hostname: str, env: str = None, logger=None):
+def init(timbermill_hostname: str, env: str = None, logger=None, static_event_params={}):
```

Comment:
same here, and let's try to minimize default values

Reply:
see my comment above

Reply:
Regarding default values, the only thing here that's mandatory is the timbermill server URL, all others are not, so I don't see how can they be minimized.
Look at my usage in pythorama, I don't see anything that we can extract to env var for example.